doc/news/changes/minor/20190307LucaHeltai

```diff
@@ -0,0 +1,4 @@
+New: Added a new function Utilities::type_to_string to demangle type names. 
```

Comment:
Please correct me if I'm wrong, but does this not have to be `Utilities::type_to_string()` for Doxygen to pick up the link to a function?